calculator.js

```diff
@@ -0,0 +1,207 @@
+var writes = document.getElementById('screen');
+var currNumber = 0;
+var result = 0;
+
+var buttons = document.querySelector('.allButtons');
+var numberButtons = buttons.querySelectorAll('.number');
+
+for (var i = 0; i <= numberButtons.length - 1; i++) {
+
+    numberButtons[i].onclick = function() {
+
+        var buttonNumber = parseInt(this.innerHTML);
+
+        currNumber = currNumber * 10 + buttonNumber
+
+        writes.value = currNumber;
+
+    }
+};
+
+var operationButtons = buttons.querySelectorAll('.operation');
+
+for (var i = 0; i <= operationButtons.length - 1; i++) {
+    var action = 1;
+    var prevOperation;
+    operationButtons[i].onclick = function() {
+
+        var operation = this.innerHTML;
+
+
+
+        /*===================================================================*/
+        if (operation === '+') {
```

Comment:
Focus to remove action for anything except the initial action. Here is a small hint:
```
if (operation === '+') {

  if (isInitialAction) {

    isInitialAction = false;
    result = currNumber;
    currNumber = 0;
    writes.value = result;

  } else {

    if (prevOperation === 'addition') {
      result = result + currNumber;
      currNumber = 0;
      writes.value = result;
    }

  }

  prevOperation = 'addition';

}